### PR TITLE
set app to json only

### PR DESCRIPTION
**Pull request by rlcheng**
Not sure how to include tests for this one so this PR comes with no tests. Existing tests still pass however.

### CODE REVIEW

**Comment by charliemcelfresh** [comment]
@rlcheng This looks pretty good, but I was looking for something more along these lines:

http://stackoverflow.com/questions/21978580/rails-4-respond-only-to-json-and-not-html

**Comment by rlcheng** [comment]
Just updated with recommendations from the link.

**Comment by charliemcelfresh** [comment]
@rlcheng Do we still need this responders gem, and that respond_to :json call, or is that code moot now ?

**Comment by rlcheng** [comment]
That code is moot. updated again. After reading up some more on respond_to, that would work best if I am trying to select a display format, like say switch between html and json.

**Comment by charliemcelfresh** [comment]
@rlcheng K -- now the tests you should write would be along the lines of making a successful JSON request, and then making a failing XML request, and a failing HTML request.

**Comment by rlcheng** [comment]
Wrote tests to get '/jokes' and tested with json, xml, and html formats. Xml and html formats return a 406 status while json passes. With the changes to do a get request for jokes in postman or curl you must now send 'Accept' 'application/json' in the header/value.

I tried forcing post request for the session test to be json format but that ends up breaking. sessioncontroller ends up rendering '400'. This has to be with the way the params are passed in so I left that alone. Same impact for the users controller.

File: config/routes.rb [hunk 1]
```diff
@@ -1,8 +1,7 @@
 Rails.application.routes.draw do
```
Comment (charliemcelfresh) [inline config/routes.rb]: @rlcheng I think the spacing on this file was fine, no need to change it

Comment (rlcheng) [inline config/routes.rb]: Just saw that. oops. I initially had only defaults: so I was trying to not repeat myself and tried a format: :json do ... end. That didn't work out and I deleted it, but forgot to untab.

**Comment by charliemcelfresh** [comment]
:hamburger: Aside from 1) catching this ticket up to master, 2) the formatting problem I mentioned

**Pull request closed without merging**